app/assets/stylesheets/components/_offices_table.scss

```diff
@@ -0,0 +1,27 @@
+.offices-table {
+  width: 100%;
+
+  .offices-table__address {
```

Comment:
Should these be nested? I thought the BEM way was not use naming (as you have) rather than nesting?